resources/js/Components/Form/CampaignCreateForm.tsx

```diff
@@ -108,8 +116,14 @@ const CampaignCreateForm = ({
             </Grid>
             <Grid container spacing={4}>
                 <Grid item xs={4}>
-                    <Button onClick={() => onCloseDrawer && onCloseDrawer()} fullWidth>
-                        Cancel
+                    <Button
+                        fullWidth
+                        onClick={() =>
+                            type === 'Create'
+                                ? window.history.back()
```

Comment:
use `Link` instead, only use `window.history.back()` for entry/dm entry since they have different entry points